tools/docker/start.sh

```diff
@@ -1,14 +1,24 @@
 #!/bin/bash
+set -ex
 
-cd web-platform-tests
-git pull --depth=50
+REPO=${1:-origin}
+BRANCH=${2:-master}
+REV=${3:-master}
```

Comment:
I think FETCH_HEAD or origin/master would be better here. Otherwise, when you get rid of the "build" branch as I've suggested, the checkout step will sometimes create a new branch called master and sometimes not.